src/track.c

```diff
@@ -760,9 +760,9 @@ float cn0_est(cn0_est_state_t *s, float I, float Q)
   return s->log_bw - 10.f*log10f(s->nsr);
 }
 
-void calc_navigation_measurement(u8 n_channels, const channel_measurement_t *meas[],
-                                 navigation_measurement_t *nav_meas[],
-                                 double nav_time, const ephemeris_t* e[])
+s8 calc_navigation_measurement(u8 n_channels, const channel_measurement_t *meas[],
```

Comment:
I'd say add some comments on the return values, but https://github.com/swift-nav/libswiftnav/pull/334 already adds doxygen for the whole thing, so I suppose this is fine here.